code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -123,3 +123,9 @@
 /mob/living/silicon/flash_act(intensity = 1, override_blindness_check = 0, affect_silicon = 0, visual = 0, type = /obj/screen/fullscreen/flash/static)
 	if(affect_silicon)
 		return ..()
+
+/mob/living/silicon/adjustToxLoss(amount, update_health, forced)
+	return adjustFireLoss(amount, update_health, forced)
```

Comment:
actually I think I'd prefer making tox damage be also healed by wires/treated like it's fire damage, rather than doing this+the below, since it might mess with some things to do this.